[Android Translate Showcase] - Crash when press back

The app crashes after opening the app and pressing back.

OS: Android 8.0.0
Device: HTC Desire 12+

Steps to reproduce:
1. Open the app with camera permission granted
2. Press back to exit the app

Logcat:
```
2020-07-12 00:19:46.854 10407-10407/com.google.mlkit.showcase.translate E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.google.mlkit.showcase.translate, PID: 10407
    java.lang.RuntimeException: Unable to destroy activity {com.google.mlkit.showcase.translate/com.google.mlkit.showcase.translate.MainActivity}: java.util.concurrent.RejectedExecutionException: Task com.google.mlkit.common.sdkinternal.zzq@e53958c rejected from java.util.concurrent.ThreadPoolExecutor@6b2a4ea[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 9]
        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4441)
        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4459)
        at android.app.ActivityThread.-wrap5(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1657)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6565)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
     Caused by: java.util.concurrent.RejectedExecutionException: Task com.google.mlkit.common.sdkinternal.zzq@e53958c rejected from java.util.concurrent.ThreadPoolExecutor@6b2a4ea[Shutting down, pool size = 1, active threads = 1, queued tasks = 0, completed tasks = 9]
        at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2078)
        at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:843)
        at java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1389)
        at com.google.mlkit.common.sdkinternal.TaskQueue.zza(com.google.mlkit:common@@16.0.0:13)
        at com.google.mlkit.common.sdkinternal.TaskQueue.submit(com.google.mlkit:common@@16.0.0:11)
        at com.google.mlkit.common.sdkinternal.ModelResource.unpin(com.google.mlkit:common@@16.0.0:7)
        at com.google.mlkit.nl.languageid.LanguageIdentifierImpl.close(com.google.mlkit:language-id@@16.1.0:38)
        at com.google.mlkit.showcase.translate.main.MainViewModel.onCleared(MainViewModel.kt:44)
        at androidx.lifecycle.ViewModel.clear(ViewModel.java:139)
        at androidx.lifecycle.ViewModelStore.clear(ViewModelStore.java:62)
        at androidx.fragment.app.FragmentManagerViewModel.clearNonConfigState(FragmentManagerViewModel.java:169)
        at androidx.fragment.app.FragmentStateManager.destroy(FragmentStateManager.java:490)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1296)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
        at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
        at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
        at androidx.fragment.app.FragmentManager.dispatchDestroy(FragmentManager.java:2609)
        at androidx.fragment.app.FragmentController.dispatchDestroy(FragmentController.java:330)
        at androidx.fragment.app.FragmentActivity.onDestroy(FragmentActivity.java:365)
        at androidx.appcompat.app.AppCompatActivity.onDestroy(AppCompatActivity.java:233)
        at android.app.Activity.performDestroy(Activity.java:7177)
        at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1249)
        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:4428)
        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4459) 
        at android.app.ActivityThread.-wrap5(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1657) 
        at android.os.Handler.dispatchMessage(Handler.java:105) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6565) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767) 
```

--- a/android/translate-showcase/app/src/main/java/com/google/mlkit/showcase/translate/main/MainFragment.kt
+++ b/android/translate-showcase/app/src/main/java/com/google/mlkit/showcase/translate/main/MainFragment.kt
@@ -81,8 +81,10 @@ class MainFragment : Fragment() {
     private lateinit var container: ConstraintLayout
     private lateinit var viewFinder: PreviewView
 
-    /** Blocking camera operations are performed using this executor */
+    /** Blocking camera and inference operations are performed using this executor. */
     private lateinit var cameraExecutor: ExecutorService
+
+    /** UI callbacks are run on this executor. */
     private lateinit var scopedExecutor: ScopedExecutor
 
     override fun onCreateView(
@@ -95,8 +97,8 @@ class MainFragment : Fragment() {
     override fun onDestroyView() {
         super.onDestroyView()
 
-        // Shut down our background executor
-        cameraExecutor.shutdown()
+        // Shut down the scoped executor. The camera executor will automatically shut down its
+        // background threads after 60s of idling.
         scopedExecutor.shutdown()
     }
 

